src/client/app/translations/data.ts

```diff
@@ -503,34 +504,34 @@ const LocaleTranslationData = {
 	},
 	"fr": {
 		"3D": "3D",
-		"400": "400 Bad Request\u{26A1}",
+		"400": "400 Bad Request",
```

Comment:
I am unclear on why you removed the special symbol at the end of untranslated strings and seem not to have done for all. Please see the PR that created them and then let me know if you think they should still be removed.